lighthouse-core/scripts/build-report-for-autodeployment.js

```diff
@@ -11,15 +11,25 @@
 const fs = require('fs');
 const path = require('path');
 const swapLocale = require('../lib/i18n/swap-locale.js');
-const ReportGenerator = require('../../report/report-generator.js');
+const buildreport = require('../../build/build-report.js');
 const {defaultSettings} = require('../config/constants.js');
-const lighthouse = require('../index.js');
+
+/** @type {import('../index.js')} */
+let lighthouse;
+
 const lhr = /** @type {LH.Result} */ (require('../../lighthouse-core/test/results/sample_v2.json'));
 const {LH_ROOT} = require('../../root.js');
 
 const DIST = path.join(LH_ROOT, `dist/now`);
 
 (async function() {
+  // Must build before importing report-generator (or indirectly through lighthouse)
+  await buildreport.buildStandaloneReport();
```

Comment:
why remove `yarn build-report && ` from `yarn now-build` ? it's simpler than doing the imports here in the right order.

the CLI options could be expanded to build psi report on a flag

Reply:
hmm..  i don't know what you do, but if i'm working on the report this is my `watch` equivalent:

```sh
find report | entr node lighthouse-core/scripts/build-report-for-autodeployment.js
```

i heart [entr](http://eradman.com/entrproject/).  (but i can't  && in that command or whatever so... the combo of the two commands is a problem).

i guess an alternative would be a new `yarn build-dist-report` script that does build-report followed by this.   that'd be fine.

wdyt?

Reply:
> but i can't && in that command or whatever so.

Do it like this:

```
find report | entr bash -c "ls && echo hi"
```